client/src/components/Ticketing/ticketingmanager/dashboard.tsx

```diff
@@ -50,6 +50,16 @@ const Dashboard = () => {
 
             </div>
           </a>
+          <a onClick={() => navigate('/ticketing/purchaseticket')} className="hover:scale-105 md:h-60 md:w- transition duration-300 ease-in-out border-none  bg-gradient-to-r from-green-500 to-zinc-500 rounded-3xl  shadow-xl  ">
```

Comment:
Do you think you could remove these changes, and then merge `main` into your branch to resolve those merge conflicts?

Reply:
These changes appear to be of your older ticket.